### PR TITLE
Add open-source Tauri app to the list

### PR DESCRIPTION
**Pull request by zhanglun**
This is the link to the project: [Lettura](https://github.com/zhanglun/lettura)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [x] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Padloc](https://github.com/padloc/padloc) - Modern, open source password manager for individuals and teams.
 - [Ghorbu Wallet](https://github.com/matthias-wright/ghorbu-wallet) - Cross-platform desktop HD wallet for Bitcoin.
 - [Stable Diffusion Buddy](https://github.com/breadthe/sd-buddy) - Desktop UI companion for the self-hosted Mac version of Stable Diffusion.
+- [Lettura](https://github.com/zhanglun/lettura) - Open-source feed reader for macOS, maybe for Windows/Linux in someday.
```
Comment (JonasKruckenberg) [inline README.md]:
```suggestion
- [Lettura](https://github.com/zhanglun/lettura) - Open-source feed reader for macOS.
```

When Linux and Windows are not yet supported it's a little misleading to advertise it using those words. Even though you explained it's not supported yet, that adds little information to readers while possibly confusing some.

Alternative just leave out all mention of OS support:
```suggestion
- [Lettura](https://github.com/zhanglun/lettura) - Open-source feed reader.
```

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]

**Comment by JonasKruckenberg** [comment]
Btw, very nice to see you use the allowlist so well 😁 great work 👍

**Comment by JonasKruckenberg** [comment]
Btw, I see you want use the allowlist. The way you have it configured right now just allows any APIs indiscriminately.

```json5
"allowlist": {
      "all": true, // this just means "allow every API"
      "http": {
        "all": true,
        "request": true,
        "scope": ["https://**", "http://**"]
      },
      "fs": {
        "all": true // do you actually need *every* filesystem API in your app?
      }
}
```

I would encourage you to revisit the allowlist and tighten it up, it's good for security and even better for app size!

**Comment by zhanglun** [comment]
> Btw, I see you want use the allowlist. The way you have it configured right now just allows any APIs indiscriminately.
> 
> ```js
> "allowlist": {
>       "all": true, // this just means "allow every API"
>       "http": {
>         "all": true,
>         "request": true,
>         "scope": ["https://**", "http://**"]
>       },
>       "fs": {
>         "all": true // do you actually need *every* filesystem API in your app?
>       }
> }
> ```
> 
> I would encourage you to revisit the allowlist and tighten it up, it's good for security and even better for app size!

thank you for your advice! I rechecked my project and made some changes. Please check [this ](https://github.com/zhanglun/lettura/blob/master/src-tauri/tauri.conf.json#L14)

**Comment by JonasKruckenberg** [comment]
```json5
"http": {
        "all": true, // this one is still the same as all: true but for the http api
        "request": true,
        "scope": ["https://**", "http://**"]
      },

```

**Comment by zhanglun** [comment]
> ```js
> "http": {
>         "all": true, // this one is still the same as all: true but for the http api
>         "request": true,
>         "scope": ["https://**", "http://**"]
>       },
> ```

ok, I modified it again, you can see [here](https://github.com/zhanglun/lettura/blob/master/src-tauri/tauri.conf.json#L14). Is there anything else to modify?

**Comment by JonasKruckenberg** [comment]
Awesome! 😁 If you're serious about security I would recommend setting a csp, currently yours is disabled (the `"csp": null` line). But I feel comfortable approving this PR as-is. Thank you for taking the time to respond to my feedback!

**Review by JonasKruckenberg — APPROVED** [review]

**Comment by zhanglun** [comment]
> Awesome! 😁 If you're serious about security I would recommend setting a csp, currently yours is disabled (the `"csp": null` line). But I feel comfortable approving this PR as-is. Thank you for taking the time to respond to my feedback!

I appreciate your advice, I will rethink the rules about security

**Pull request merged**